### ISSUE TITLE
Redshift thinks the sun rises at 22:00

### ISSUE THREAD

**Issue by nweston** [jonls/redshift#297]
I've noticed that redshift always seem to be in "night" mode on my system. So I added some code to print out the solar elevation and period at different times of day. This is what I got:

Location: 42.36 N, 71.06 E
Temperatures: 5700K at day, 3500K at night
Solar elevations: day above 3.0, night below -6.0
Brightness: 1.00:1.00
Gamma (Daytime): 1.000, 1.000, 1.000
Gamma (Night): 1.000, 1.000, 1.000
Status: Enabled
GMT 0:00, local 19:00, time 1452297600.000000, elevation -29.7022, period: Night
GMT 1:00, local 20:00, time 1452301200.000000, elevation -18.7567, period: Night
GMT 2:00, local 21:00, time 1452304800.000000, elevation -8.22346, period: Night
GMT 3:00, local 22:00, time 1452308400.000000, elevation 1.59893, period: Transition
GMT 4:00, local 23:00, time 1452312000.000000, elevation 10.3477, period: Daytime
GMT 5:00, local 0:00, time 1452315600.000000, elevation 17.5652, period: Daytime
GMT 6:00, local 1:00, time 1452319200.000000, elevation 22.7115, period: Daytime
GMT 7:00, local 2:00, time 1452322800.000000, elevation 25.2635, period: Daytime
GMT 8:00, local 3:00, time 1452326400.000000, elevation 24.9036, period: Daytime
GMT 9:00, local 4:00, time 1452330000.000000, elevation 21.6793, period: Daytime
GMT 10:00, local 5:00, time 1452333600.000000, elevation 15.9773, period: Daytime
GMT 11:00, local 6:00, time 1452337200.000000, elevation 8.33987, period: Daytime
GMT 12:00, local 7:00, time 1452340800.000000, elevation -0.707903, period: Transition
GMT 13:00, local 8:00, time 1452344400.000000, elevation -10.7322, period: Night
GMT 14:00, local 9:00, time 1452348000.000000, elevation -21.3894, period: Night
GMT 15:00, local 10:00, time 1452351600.000000, elevation -32.3878, period: Night
GMT 16:00, local 11:00, time 1452355200.000000, elevation -43.4248, period: Night
GMT 17:00, local 12:00, time 1452358800.000000, elevation -54.0593, period: Night
GMT 18:00, local 13:00, time 1452362400.000000, elevation -63.3782, period: Night
GMT 19:00, local 14:00, time 1452366000.000000, elevation -69.184, period: Night
GMT 20:00, local 15:00, time 1452369600.000000, elevation -68.2902, period: Night
GMT 21:00, local 16:00, time 1452373200.000000, elevation -61.329, period: Night
GMT 22:00, local 17:00, time 1452376800.000000, elevation -51.5544, period: Night
GMT 23:00, local 18:00, time 1452380400.000000, elevation -40.7609, period: Night

In short, the night->day transition begins around 22:00 local time, and the day->night transition around 7:00 local time. This seems to be off by about 9 hours (current sunrise/sunset times are 7:15 and 16:30 here).

I'm seeing the same behavior with 1.8 (installed from the Ubuntu package) and master build from source.

**Comment by jonls** [comment]
Hi @nwesten, do you live in Kyrgyzstan? I'm guessing that you probably live around Boston which is west of the prime meridian not east. As you can see in the output, it shows 71.06 E longitude. If you specify the location manually you simply have to set your location as 42.36, -71.06 (with negative longitude).

**Comment by nweston** [comment]
Yup. Cut-and-paste error when I entered the location in the config file. Sorry about that, and thanks for the quick response.
